personhood/contract_credential.go

```diff
@@ -29,24 +36,46 @@ type ContractCredential struct {
 	CredentialStruct
 }
 
+// VerifyInstruction allows for an unsigned "recover" command that will be verified later.
+func (c ContractCredential) VerifyInstruction(rst byzcoin.ReadOnlyStateTrie, inst byzcoin.Instruction, ctxHash []byte) error {
+	if inst.Invoke != nil && inst.Invoke.Command == "recover" {
```

Comment:
Can we have more details here why this particular case needs to skip the verification ?